### DISCUSSION TITLE
Can i set up PEACOCK server on my host?

### DISCUSSION THREAD

**Discussion by Undore** [thepeacockproject/Peacock#475] [Q&A] [answered]
Not localy, on linux virtual machine for example. I want it to be accessable not only from my PC, but also from other PCs,  saving all progress

**Comment by AnthonyFuller** [comment, marked as answer]
You can certainly host it publicly, although we currently do not endorse doing so.

EDIT: You can use the Linux version of Peacock if you install the correct version of Node.
